src/PageHeader/PageHeader.tsx

```diff
@@ -58,13 +58,24 @@ export type PageHeaderProps = {
 
 const Root: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({children, sx = {}, as = 'div'}) => {
   const rootStyles = {
-    display: 'flex',
-    flexDirection: 'column',
+    display: 'grid',
+    // We have max 4 columns.
+    gridTemplateColumns: 'auto auto auto 1fr',
+    // First column is for context area and takes the full row. Context area items are displayed with flexbox inside.
```

Comment:
I'm a little confused. The context area is a row, not a column. Is there something else that goes in the first column?

Reply:
Since you switched to using grid-template-areas, I don't think we need these comment blocks. The code documents itself! 😄

Reply:
Yeah you are right. It rather confuses I guess 😄 I'll remove them